source/stylesheets/modules/_app-pane.scss

```diff
@@ -26,6 +26,7 @@
       .app-pane__body {
         display: flex;
         flex: 1 1 100%;
+        min-height:0;
```

Comment:
This is _really really really_ finicky but can we make this `min-height: 0;` (with a space after the colon) for consistency?